cluster/gce/gci/configure-helper.sh

```diff
@@ -92,8 +94,10 @@ EOF
     missingok
     notifempty
     compress
-    size 100M
+    maxsize 100M
     daily
+    dateext
+    dateformat -%Y%m%d-%s
```

Comment:
Yeah. From my testing, this is the only combination (maxsize with dateformat) that gives us expected behavior.